src/sentry/static/sentry/app/views/usageStats/index.tsx

```diff
@@ -1,63 +1,94 @@
 import React from 'react';
-import {RouteComponentProps} from 'react-router';
-import moment from 'moment';
+import {browserHistory, RouteComponentProps} from 'react-router';
 
+import {DateTimeObject} from 'app/components/charts/utils';
 import PageHeading from 'app/components/pageHeading';
+import {DEFAULT_RELATIVE_PERIODS, DEFAULT_STATS_PERIOD} from 'app/constants';
 import {t, tct} from 'app/locale';
 import {PageContent, PageHeader} from 'app/styles/organization';
-import {DataCategory, Organization} from 'app/types';
+import {DataCategory, Organization, RelativePeriod} from 'app/types';
 
-import {OrganizationUsageStats, ProjectUsageStats} from './types';
+import {ChartDataTransform} from './usageChart';
 import UsageStatsOrg from './usageStatsOrg';
-import UsageStatsProjects from './usageStatsProjects';
+// import UsageStatsProjects from './usageStatsProjects';
 
 type Props = {
   organization: Organization;
-  orgStatsLoading: boolean;
-  projectStatsLoading: boolean;
-  orgStats?: OrganizationUsageStats;
-  orgStatsError?: Error;
-  projectStats?: ProjectUsageStats[];
-  projectStatsError?: Error;
 } & RouteComponentProps<{orgId: string}, {}>;
 
-type State = {
-  dataCategory: DataCategory;
-  dateStart: moment.Moment;
-  dateEnd: moment.Moment;
-};
-
-class OrganizationStats extends React.Component<Props, State> {
-  state: State = {
-    dataCategory: DataCategory.ERRORS,
-    dateStart: moment().subtract(14, 'days'),
-    dateEnd: moment(),
-  };
+class OrganizationStats extends React.Component<Props> {
+  get dataCategory(): DataCategory {
+    const dataCategory = this.props.location?.query?.dataCategory;
 
-  setDataCategory = (dataCategory: DataCategory) => {
-    this.setState({dataCategory});
-  };
+    switch (dataCategory) {
+      case DataCategory.ERRORS:
+      case DataCategory.TRANSACTIONS:
+      case DataCategory.ATTACHMENTS:
+        return dataCategory as DataCategory;
+      default:
```

Comment:
again should make sure sessions handled correctly

Reply:
This reads from the URL bar and the user can either amend it directly or use the dropdown selector (which has only errors/transactions/attachments), so I prefer to default to errors here.